monkey/monkey_island/cc/ui/src/components/utils/ServerUtils.tsx

```diff
@@ -67,3 +67,21 @@ export function getEventSourceHostname(event_source, agents, machines): string {
 
   return hostname;
 }
+
+export function getManualMonkeysHostnames(agents, machines){
```

Comment:
There should be a method `getManuallyStartedAgents`. This method could just filter the agents. Then another method could be `getAgentMachine`. This would make sure that both methods are reusable and the same goal could be achieved similarly: 
```
let hostnames = getManuallyStartedAgents(agents).map((agent) => getAgentMachine(agent, machines).hostname);
```